Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:2.7.12
+EXPOSE 5000
+VOLUME /root/.octoprint
+
+RUN  wget -O ffmpeg.tar.xz https://johnvansickle.com/ffmpeg/releases/ffmpeg-release-32bit-static.tar.xz && mkdir -p /opt/ffmpeg && tar xvf ffmpeg.tar.xz -C /opt/ffmpeg --strip-components=1
+
+ADD ./ /opt/octoprint/
+RUN cd /opt/octoprint && python setup.py install
```

Comment:
If I understand this correctly (sorry, no experience with Docker myself), this will basically copy the sources to `/opt/octoprint` and then run a global install? That way you'll get OctoPrint twice - once as a pure source checkout and once installed "globally" (well, container wide I guess?) in Python's site-packages (not under `/opt` I think). The copy under `/opt/` will have basically no function. I'm not sure this is the best/cleanest approach here, especially since the official installation instructions pretty much mandate the usage of a virtual environment. Yes, I know, in this containerized setup that currently is not necessary, but if this file is to be merged into the official source repository (and I have to admit I'm still not entirely sure whether that's the best approach and if it wouldn't rather make sense to put that into a different repo that then pulls in the source tree) it should not go against all recommendations in the docs.